buildSrc/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -126,6 +124,15 @@ tasks {
     }
   }
 
+  named("jar", Jar::class.java) {
```

Comment:
Can you add a comment indicating that this adds required OSGi metadata to the jar manifest?